packages/expo-modules-core/android/src/main/java/expo/modules/core/utilities/EmulatorUtilities.kt

```diff
@@ -0,0 +1,32 @@
+package expo.modules.core.utilities
+
+import android.os.Build
+import java.util.*
+
+object EmulatorUtilities {
+  fun isRunningOnEmulator(): Boolean {
```

Comment:
nit: add a comment indicating where this is copied/derived from (the link to react-native-device-info should be fine)

Reply:
Well-appreciated nit! and done!